scripts/local-universe.py

```diff
@@ -319,35 +356,36 @@ def add_http_resource(dir_path, url, base_path):
 
 
 def prepare_repository(
-    package, package_path,
-    source_repo, dest_repo,
+    package,
+    version,
+    package_path,
+    source_repo,
+    dest_repo,
     http_root,
     skip_images,
     skip_cli
 ):
     dest_path = dest_repo / package_path.relative_to(source_repo)
     shutil.copytree(str(package_path), str(dest_path))
 
-    source_resource = package_path / 'resource.json'
     dest_resource = dest_path / 'resource.json'
-    with source_resource.open(encoding='utf-8') as source_file, \
-            dest_resource.open('w', encoding='utf-8') as dest_file:
-        resource = json.load(source_file)
+    with dest_resource.open('w', encoding='utf-8') as dest_file:
+        resource = load_json(package_path / 'resource.json')
 
         # Change the root for images (ignore screenshots)
         if not skip_images and 'images' in resource:
             resource["images"] = {
                 n: urllib.parse.urljoin(
                     http_root, str(pathlib.PurePath(
-                        package, "images", pathlib.Path(uri).name)))
+                        package, version, "images", pathlib.Path(uri).name)))
```

Comment:
Yeah. Unfortunately, we do that in two places. In general, the functional separation of this script is not that great. I think it is okay for now since we plan to deprecate this script after 1.11.